packaging/standalone/src/tests/shell-scripts/test-console.sh

```diff
@@ -9,4 +9,9 @@ test_expect_success "should run okay" "
   run_console
 "
 
+test_expect_success "should run okay with env var" "
```

Comment:
Shouldn't the test name here specify that it's an env var starting with a number?